aea/channel/local.py

```diff
@@ -128,17 +124,6 @@ def handle_agent_message(self, envelope: Envelope) -> None:
         else:
             self._send(envelope)
 
-    def register_agent(self, public_key: str, agent_description: Description) -> None:
```

Comment:
Why are the register and unregister method being removed?

Reply:
`register_agent` and `unregister_agent` methods are not supported anymore:
https://github.com/fetchai/oef-sdk/blob/feat/python-sdk2/oef/src/python/agents.py#L521-L546

Reply:
Ok, thanks, I saw this thread on Slack now.